lang/en/docs/materials-designer/source-editor/basis.md

```diff
@@ -60,6 +60,15 @@ Click on the animation below to see the above in action. Here, we change the x-a
 
 <img data-gifffer="/images/materials-designer/ChangeMaterialBasis.gif" />
 
+
+## Non-Periodic Basis Transformations
+When [Edit - Toggle "isNonPeriodic"](../header-menu/edit.md) is used, the basis of the structure will automatically be converted to Cartesian Coordinates and translated to the center of the lattice. This is done by translating each basis coordinate so that the center of coordinates for the structure and the center point of the lattice align.
+
```

Comment:
We should refer to the change in lattice in https://github.com/Exabyte-io/documentation/pull/219/files#diff-19436c43ea51eedc5797131c91c17f24ac0330170499be4db6563c2e9a64b953R41 then